wallet/src/client.rs

```diff
@@ -38,6 +40,10 @@ pub fn create_coinbase(url: &str, block_fees: &BlockFees) -> Result<CbData, Erro
 				"Failed to get coinbase from {}. Run grin wallet listen",
 				url
 			);
+			has_error = true;
+		}
+		if (has_error == true) {
```

Comment:
Sorry, just a further nit-pick, parentheses aren't necessary in rust if conditions, so just
````
if has_error
````
should suffice here. I only say so because the way its written will add an unnecessary warning to the build output.

Reply:
done. I also removed the {} since it is a single line if condition.